gnomad/utils/vep.py

```diff
@@ -388,7 +400,8 @@ def filter_vep_to_synonymous_variants(
 
 
 def vep_struct_to_csq(
-    vep_expr: hl.expr.StructExpression, csq_fields: str = VEP_CSQ_FIELDS
+    vep_expr: hl.expr.StructExpression,
+    csq_fields: str = VEP_CSQ_FIELDS[CURRENT_VEP_VERSION],
```

Comment:
I can't add it on L419, but can you add something like `Default is the CSQ fields of the CURRENT_VEP_VERSION.`

Reply:
I did, sorry, I will keep in mind changing the docs every time I change the function.